source/utils/dirent.h

```diff
@@ -0,0 +1,1166 @@
+/*
+ * Dirent interface for Microsoft Visual Studio
```

Comment:
Why is this here?

Reply:
It was in ultils before I’ll just delete it